### PR TITLE
LPS-181043

### PR DESCRIPTION
**Pull request by danvb112**
Related Issues:
https://issues.liferay.com/browse/LPS-180665
https://issues.liferay.com/browse/LPS-180718
https://issues.liferay.com/browse/LPS-181043

### CODE REVIEW

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
Just started reviewing :)

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/assets/icons/business-center.svg
Comment (naraandrade) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/assets/icons/business-center.svg:1]: @danvb112 Please, avoid using svg files. You can use png files instead.

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Comment by naraandrade** [comment]
Resent here: https://github.com/brianchandotcom/liferay-portal/pull/133166

**Pull request closed without merging**